config/tls.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+)
+
+func TLS(certPath string, keyPath string, clientCAPath string) (*tls.Config, error) {
+
```

Comment:
Style: unnecessary newline

Reply:
Sure. these blank lines is our internal coding style, it's hard for me not add any :)